python/adelphi/bin/adelphi

```diff
@@ -2,39 +2,60 @@
 
 import argparse
 import logging
+import os
+import os.path
 from functools import partial
 
+try:
+    from tempfile import TemporaryDirectory
+except ImportError:
+    from backports.tempfile import TemporaryDirectory
+
```

Comment:
Context mgr version of temp management not available on py2k so we swap in a backport in that situation